spec/lib/report/account_events_spec.rb

```diff
@@ -0,0 +1,70 @@
+RSpec.describe Report::AccountEvents do
+  let(:report) { described_class.report(start_date: start_date, end_date: end_date, user_id_pepper: "pepper") }
+
+  let(:start_date) { Time.zone.local(2020, 1, 22, 10, 0, 0) }
+  let(:end_date) { start_date + 1.day }
+
+  context "with no login events" do
```

Comment:
Does report get all the events?
All new accounts should start with an account created event now.

There are a few folks who both will have created an account before we added the initial event and never have logged in.

What this test just trying to show it doesn't error for now data?

Reply:
It specifically gets the login events, and yes this test is just to check that it returns an empty report if there's nothing there.